src/lib/spinel/radio_spinel.hpp

```diff
@@ -1084,6 +1088,10 @@ class RadioSpinel
      *
      */
     otError VendorHandleValueIs(spinel_prop_key_t aPropKey);
+
+    void SetVenderRestorePropertiesCallback(
+        otRadioSpinelVenderRestorePropertiesCallback aVenderRestorePropertiesCallback,
+        void                                        *aVenderRestorePropertiesContext);
```

Comment:
Can consider using shorter names for the inputs?
```suggestion
        otRadioSpinelVenderRestorePropertiesCallback aCallback, void *aContext);
```